change/@fluentui-react-charting-4b4305e3-4ff3-49f0-88b1-d147b5063a05.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30256&commitId=20cb63e59678a4879beae76f12e36d0a6c6460c5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- centerCover, image smaller.default.chromium.png|0| Removed|
|Image.Fit- centerCover, image taller.default.chromium.png|0| Removed|
|Image.Fit- centerCover, image wider.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>